invenio_oaiserver/utils.py

```diff
@@ -29,13 +29,39 @@
 from functools import partial
 
 from flask import current_app
+from lxml import etree
+from lxml.builder import E
+from lxml.etree import Element
 from werkzeug.utils import import_string
 
 try:
     from functools import lru_cache
 except ImportError:  # pragma: no cover
     from functools32 import lru_cache
 
+ns = {
+    None: 'http://datacite.org/schema/kernel-4',
+    'xsi': 'http://www.w3.org/2001/XMLSchema-instance',
+    'xml': 'xml',
+}
+
+NS_EPRINTS = {None: 'http://www.openarchives.org/OAI/1.1/eprints'}
+EPRINTS_SCHEMA_LOCATION = 'http://www.openarchives.org/OAI/1.1/eprints'
+EPRINTS_SCHEMA_LOCATION_XSD = \
+    'http://www.openarchives.org/OAI/1.1/eprints.xsd'
```

Comment:
We should maybe be consistent and use the 2.0 definition of eprints, which actually does not differ from 1.1:

https://www.openarchives.org/OAI/1.1/eprints.xsd
https://www.openarchives.org/OAI/2.0/eprints.xsd

Reply:
Interestingly enough the [guidelines](http://www.openarchives.org/OAI/2.0/guidelines-eprints.htm) are also inconsistent.